Mike Hulbert/week_01/calculator/exercise.js

```diff
@@ -0,0 +1,67 @@
+console.log("is this on");
+
+// # The Calculator
+//
+// ## Part 1
+// - Write a function called squareNumber that will take one argument (a number), square that number, and return the result. It should also log a string like "The result of squaring the number 3 is 9."
+// - Write a function called halfNumber that will take one argument (a number), divide it by 2, and return the result. It should also log a string like "Half of 5 is 2.5.".
+// - Write a function called percentOf that will take two numbers, figure out what percent the first number represents of the second number, and return the result. It should also log a string like "2 is 50% of 4."
+// - Write a function called areaOfCircle that will take one argument (the radius), calculate the area based on that, and return the result. It should also log a string like "The area for a circle with radius 2 is 12.566370614359172."
+// - Bonus: Round the result so there are only two digits after the decimal.
+//
+
+// Function for calculating the square root ------------------------------------
+const squareFunction = function(num1) {
+  const result = num1 * num1;
+  console.log(`The result of squaring the number ${num1} is ${result}.`)
+  return result;
+};
+
+squareFunction(3);
+
+// Function for dividing a number by 2 -----------------------------------------
+const halfNumber = function(num1) {
+  const result = num1 / 2;
+  console.log(`Half of ${num1} is ${result}.`);
+  return result;
+};
+
+halfNumber(5);
+
+// Function for calculating the percentage -------------------------------------
+const percentOf = function(num1, num2){
```

Comment:
minor thing, for readability this is better to be : 
`const percentOf = function (num1, num2) {`